label_studio/tasks/api.py

```diff
@@ -181,18 +180,21 @@ def get_retrieve_serializer_context(self, request):
 
     def get(self, request, pk):
         context = self.get_retrieve_serializer_context(request)
-        context['project'] = project = self.task.project
+        context['project'] = self.task.project
 
         # get prediction
-        if (
-            project.evaluate_predictions_automatically or project.show_collab_predictions
-        ) and not self.task.predictions.exists():
```

Comment:
Why do we no longer check for `project.evaluate_predictions_automatically`?

Reply:
I have since learned that it was deprecated as part of this work - we should add this fact to the PR description